browser.js

```diff
@@ -36,7 +36,7 @@ function isEncoding (encoding) {
   try {
     codecFor(encoding)
     return true
-  } catch {
```

Comment:
this is valid javascript, is that a limitation in some parser?

Reply:
Yes, it's related to `tinyify`.

Reply:
It's really hard to find where something goes wrong in how everything fits together with `tinyify`, `minify-stream`, `envify`, `terser`. Working on finding a real fix, but takes a lot of time and debugging. This PR would help tremendously as a workaround.